src/content/1.7/modules/core_updates/1.7.7.md

```diff
@@ -0,0 +1,40 @@
+
+## Order view page layout modifications
+
+In this version the order page has been migrated with Symfony. We used this migration as an opportunity to improve the page layout to meets the merchants' needs.
+You can see a representation of these new hooks in the page here:
+
+{{< figure src="../img/order_view_page_hooks.jpg" title="Order view page hooks layout" >}}
+
+
+### Modified hooks
+
+The following hooks **have been moved** and have new names. Even though they are very similar to their former equivalents the layout has changed and thus the modules need to adapt their rendering.
+
+Former hook                                                            | New hook                        | Modification
+---------------------------------------------------------------------- |:------------------------------- |:-----------------------------------------------------------------------------------------------------------------------------------------------------------------
+**displayAdminOrderRight**                                             | **displayAdminOrderSideBottom** | It is now displayed on the **left** which is why it was renamed based on its context not its position, since the layout is completely different it has no alias
+**displayAdminOrderLeft**                                              | **displayAdminOrderMainBottom** | It is now displayed on the **right** which is why it was renamed based on its context not its position, since the layout is completely different it has no alias
+**displayAdminOrderTabOrder** and **displayAdminOrderTabShip**         | **displayAdminOrderTabLink**    | Both hooks are now combined in a single generic hook that allows you to include tab links the the order view page, it only has the *id_order* as a parameter
+**displayAdminOrderContentOrder** and **displayAdminOrderContentShip** | **displayAdminOrderTabContent** | Both hooks are now combined in a single generic hook that allows you to include tab contents the the order view page, it only has the *id_order* as a parameter
+
+These hooks have been aliased, although we recommend migrating to the new hooks and check that it integrates correctly with the new layout:
+
+Former hook                       | New aliased hook                | Modification
+--------------------------------- |:------------------------------- |:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+**displayBackOfficeOrderActions** | **displayAdminOrderSideBottom** | Although it has been aliased for convenience, the new hook is not displayed at the same place at all (it is now in the side column), so you'd better check how it integrates And if you need to integrate action buttons please use the new **actionGetAdminOrderButtons** hook
```

Comment:
```suggestion
**displayBackOfficeOrderActions** | **displayAdminOrderSideBottom** | Although it has been aliased for convenience, the new hook is not displayed at the same place at all (it is now in the side column), so you'd better check how it integrates and, if you need to integrate action buttons, please use the new **actionGetAdminOrderButtons** hook
```

Reply:
It slightly changes the sentence and it's too long I used `However` instead